service/src/main/java/edu/tamu/catalog/service/FolioCatalogService.java

```diff
@@ -713,10 +714,11 @@ private boolean getManualBlockStatus(String userId) {
                                 .year(recordValues.getYear())
                                 .genre(recordValues.getGenre())
                                 .fallbackLocationCode(fallbackLocationCode)
+                                .holdingLocation(holdingLocationName)
                                 .edition(recordValues.getEdition())
                                 .oclc(recordValues.getOclc())
                                 .recordId(recordValues.getRecordId())
-                                .callNumber(recordValues.getCallNumber())
+                                .callNumber(holding.at("/callNumber").asText())
```

Comment:
This might be cleaner to hoist up to a variable.

Reply:
Updated